server/data/data-jmap-cassandra/src/test/java/org/apache/james/jmap/cassandra/upload/CassandraUploadRepositoryTest.java

```diff
@@ -76,16 +76,4 @@ public void deleteShouldReturnFalseWhenRowDoesNotExist() {
     public UpdatableTickingClock clock() {
         return clock;
     }
-
-    @Disabled("Delete method always return true (to avoid LWT)")
-    @Override
-    public void deleteShouldReturnTrueWhenRowExists() {
```

Comment:
no, please keep it

Reply:
I doubted this also. But I guess that we removed that test in master?

Reply:
No, it still exist in master branch
https://github.com/apache/james-project/blob/master/server/data/data-jmap-cassandra/src/test/java/org/apache/james/jmap/cassandra/upload/CassandraUploadRepositoryTest.java#L64

Reply:
It is already there, messed up during the rebase, it was duplicated, thus not compiling. That's just removing the duplicate tests